Mathlib/GroupTheory/SpecificGroups/Coxeter.lean

```diff
@@ -421,7 +420,7 @@ def G₂ : Matrix (Fin 2) (Fin 2) ℕ :=
      6, 1]
 
 theorem G₂IsCoxeter : IsCoxeter G₂ where
-  symmetric := by simp [Matrix.IsSymm]; decide
+  symmetric := by unfold Matrix.IsSymm; decide
```

Comment:
If you add an instance for `DecidablePred Matrix.IsSymm` (which should be pretty trivial), you will be able to golf this even further to
```suggestion
  symmetric := by decide
```